frontend/public/components/default-resource.jsx

```diff
@@ -60,4 +60,25 @@ export const DefaultDetailsPage = props => {
   return <DetailsPage {...props} menuActions={menuActions} pages={pages} />;
 };
 
+export const DefaultOverviewPage = ({kind, resource}) => {
+  return <div className="co-m-pane resource-overview">
+    <div className="co-m-nav-title resource-overview__heading">
```

Comment:
Would be nice to pull the heading out into a shareable component since we'll need to repeat this markup a half-dozen times when we start adding more resources.

Reply:
`NavTitle` is the component we would want to use, but it is specifically designed to wrapped by `Firehose` and receives props in that format. I think we could probably make it work, but we might want to revisit `NavTitle` and see if we can make it a little more extensible.